doc/import-block-pv.md

```diff
@@ -0,0 +1,120 @@
+# How to import an image to a block raw PV User Guide
+The purpose of this document is to show how to import an image to a raw block PV.
+For now, this functionality only works for importing an image from a URL (http/s3) without authentication.
+
+## Prerequesites
+- You have a Kubernetes cluster up and running with CDI installed and at least one PersistentVolume is available.
+- Feature-Gate 'BlockVolume' is enabled.
+
+## Import an image to a Local raw block PV
+In case you do not import an image to a local PV, please skip to the next section (Import an image with DataVolume manifest).
+There are few steps to follow, to be able to import an image to a local raw block PV.
+
+First, create a local PV with the volumeMode field sets to 'Block':
+
+```bash
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: local-block-raw-import-pv
+  annotations:
+spec:
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - node01
+  volumeMode: Block  # This is Block PV
+  storageClassName: local
```

Comment:
storageClassName is optional, might want to note that if your default storageClass is a block mode, then you don't need this.